package.json

```diff
@@ -31,9 +31,9 @@
     "http-proxy-agent": "^2.1.0",
     "https-proxy-agent": "^2.2.1",
     "lru-cache": "^4.1.2",
-    "pac-proxy-agent": "^2.0.1",
+    "pac-proxy-agent": "git://github.com/liuzhen2008/node-pac-proxy-agent.git#master",
```

Comment:
No, please, don't depend on git repos and #master versions — that breaks semver.

Reply:
Oops. didn't mean to do this..

Reply:
@ChALkeR  Please see pull request https://github.com/TooTallNate/node-proxy-agent/pull/20

I submitted a pull request from a wrong branch.